mlflow/server/js/src/experiment-tracking/pages/experiment-overview/components/TraceRequestsChart.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useMemo } from 'react';
+import { useDesignSystemTheme, Typography, ChartLineIcon } from '@databricks/design-system';
+import { FormattedMessage } from 'react-intl';
+import { BarChart, Bar, XAxis, YAxis, Tooltip, ResponsiveContainer } from 'recharts';
```

Comment:
Please create a [lazy wrapper](https://react.dev/reference/react/lazy) around chart component imported from recharts and use it here. We need to facilitate code splitting, esp. if this will go to DB eventually